cohorts/plot.py

```diff
@@ -82,11 +137,16 @@ def fishers_exact_plot(data, condition1, condition2, ax=None, alternative="two-s
         x=condition1,
         y=condition2,
         ax=ax,
-        data=data
+        data=data,
+        **kwargs
     )
+
     count_table = pd.crosstab(data[condition1], data[condition2])
     print(count_table)
     oddsratio, pvalue = fisher_exact(count_table, alternative=alternative)
+    hide_negative_y_ticks(plot)
+    only_percentage_ticks(plot)
```

Comment:
Oh. Right. Looks like that's confusingly named; perhaps I was intending them to be bundled but better did, so `only_percentage_ticks` just handles <= 1. I just moved the Fisher's exact call to `hide_negative_y_ticks` into `only_percentage_ticks`.